server/api/src/rsFileChksum.cpp

```diff
@@ -349,18 +348,17 @@ int file_checksum(RsComm* _comm,
         std::string checksum_from_resource;
         std::string final_scheme_str(final_scheme);
         ret_err = resc->call<const std::string*, std::string*>(
-            _comm, irods::RESOURCE_OP_READ_CHECKSUM, file_ptr, &final_scheme_str, &checksum_from_resource);
+            _comm, irods::RESOURCE_OP_READ_CHECKSUM_FROM_STORAGE_DEVICE, file_ptr, &final_scheme_str, &checksum_from_resource);
         if (ret_err.ok()) {
             strncpy(_calculated_checksum, checksum_from_resource.c_str(), NAME_LEN);
             return 0;
         }
         log_api::trace(fmt::format(
```

Comment:
The explicit call to `fmt::format` can be removed.

Reply:
Your comment states the work was done, but `fmt::format` is still in the code?

Perhaps the change wasn't pushed?